solr/bin/solr.cmd

```diff
@@ -362,23 +362,21 @@ goto done
 @echo.
 @echo   --no-prompt   Don't prompt for input; accept all defaults when running examples that accept user input
 @echo.
-@echo   -v and -q     Verbose (-v) or quiet (-q) logging. Sets default log level to DEBUG or WARN instead of INFO
-@echo.
-@echo   -V/--verbose  Verbose messages from this script
+@echo   -d/--debug and -q/--quiet Debug (-d) or quiet (-q) logging. Sets default log level to DEBUG or WARN instead of INFO
 @echo.
 goto done
 
 :stop_usage
 @echo.
-@echo Usage: solr stop [-k key] [-p port] [-V]
+@echo Usage: solr stop [-k key] [-p port] [-d]
 @echo.
 @echo  -k key         Stop key; default is solrrocks
 @echo.
 @echo  -p port        Specify the port the Solr HTTP listener is bound to
 @echo.
 @echo  --all          Find and stop all running Solr servers on this host
 @echo.
-@echo  -V/--verbose   Verbose messages from this script
+@echo  -d/--debug     Debug messages from this script
```

Comment:
If we go in the CLI classes with `--debug` without `-d`, then we should reflect the changes here as well to avoid any inconsistencies.